### ISSUE TITLE
Usage with legacy version of sails

### ISSUE THREAD

**Issue by winrid** [balderdashy/sails-mongo#363]
While the readme indicates that this project works with sails v0.9.x when I attempt to use the new adapter (0.11.5) I get:

    2015-11-23T23:38:02.308Z - warn: The adapter `sails-mongo` appears to be designed for a later version of Sails (v0.10)
    2015-11-23T23:38:02.311Z - warn: (it has a `registerConnection()` method.)
    2015-11-23T23:38:02.311Z - warn: Since you're running Sails v0.9.17, it won't be compatible.

    2015-11-23T23:38:02.311Z - warn: To install a compatible version of `sails-mongo`, run:
    2015-11-23T23:38:02.312Z - warn: npm install sails-mongo@0.9.x

    2015-11-23T23:38:02.312Z - error: There was an error attempting to require("sails-mongo")
    2015-11-23T23:38:02.312Z - error: Is this a valid Sails/Waterline adapter?  The following error was encountered ::
    2015-11-23T23:38:02.312Z - error: Adapter `sails-mongo` is not compatible with the current version of Sails.

This is a fairly large project and can't be migrated to 0.10.x quickly but some features from this version of the adapter have been long awaited....

**Comment by sailsbot** [comment]
Thanks for posting, @winrid.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
